ludwig/data/dataset/base.py

```diff
@@ -31,6 +31,10 @@ def initialize_batcher(self, batch_size=128, should_shuffle=True, seed=0, ignore
     def to_df(self):
         raise NotImplementedError()
 
+    @property
+    def in_memory_size_bytes(self):
+        raise NotImplementedError()
```

Comment:
Can we add an implementation for dask too?

Reply:
@jppgks  believe Dask/Modin/Pandas all use the PandasDatasetManager class, and all 3 have the same API to check memory usage. I will double check just in case I missed this.